recipes/wgms3d/meta.yaml

```diff
@@ -0,0 +1,102 @@
+# Note: there are many handy hints in comments in this example -- remove them when you've finalized your recipe
+
+# Jinja variables help maintain the recipe as you'll update the version only here.
+# Using the name variable with the URL in line 14 is convenient
+# when copying and pasting from another recipe, but not really needed.
+{% set name = "wgms3d" %}
+{% set version = "2.0" %}
+{% set variant = "openblas" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+# source:
+  # url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/{{ name }}-{{ version }}.tar.gz
+  # If getting the source from GitHub, remove the line above,
+  # uncomment the line below, and modify as needed. Use releases if available:
+  # url: https://github.com/simplejson/simplejson/releases/download/{{ version }}/simplejson-{{ version }}.tar.gz
+  # and otherwise fall back to archive: 
+  # url: https://github.com/simplejson/simplejson/archive/v{{ version }}.tar.gz
+  # sha256: d58439c548433adcda98e695be53e526ba940a4b9c44fb9a05d92cd495cdd47f
+  # sha256 is the preferred checksum -- you can get it for a file with:
+  #  `openssl sha256 <file name>`.
+  # You may need the openssl package, available on conda-forge:
+  #  `conda install openssl -c conda-forge``
+
+source:
+    fn: wgms3d-{{ version }}.tar.gz
+    url: http://www.soundtracker.org/raw/wgms3d/wgms3d-{{ version }}.tar.gz
+    sha256: c68b1b868d84d3351622ec5b4df5b6af1e0ac916
```

Comment:
It seems the SHA is `cba974d049db4162904602858bb0bc2069af8c83c9360e2c9eadb1ef560ac3fa`.

Reply:
Thank you David